coredns/tests/test_check.py

```diff
@@ -0,0 +1,172 @@
+# (C) Datadog, Inc. 2010-2017
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+import os
+import mock
+import sys
+import pytest
+import requests
+import subprocess
+import time
+
+# project
+from datadog_checks.coredns import CoreDNSCheck
+from datadog_checks.dev import docker_run, RetryError
+from datadog_checks.utils.common import get_docker_hostname
+
+instance = {
+    'prometheus_endpoint': 'http://localhost:9153/metrics',
+}
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+CONFIG_FOLDER = os.path.join(HERE, 'docker', 'coredns')
+HOST = get_docker_hostname()
+PORT = '9153'
+URL = "http://{}:{}/metrics".format(HOST, PORT)
+
+DIG_ARGS = [
+    "dig",
+    "google.com",
+    "@127.0.0.1",
+    "-p",
+    "54"
+]
+
+
+class MockResponse:
+    """
+    MockResponse is used to simulate the object requests.Response commonly returned by requests.get
+    """
+
+    def __init__(self, content, content_type):
+        self.content = content
+        self.headers = {'Content-Type': content_type}
+
+    def iter_lines(self, **_):
+        for elt in self.content.split("\n"):
+            yield elt
+
+    def raise_for_status(self):
+        pass
+
+    def close(self):
+        pass
+
+
+@pytest.fixture(scope="session")
+def spin_up_coredns():
```

Comment:
Can you please put all these fixtures in a `conftest.py`? At that point they will be globally available.